### ISSUE TITLE
More advanced starter sites listing options

### ISSUE THREAD

**Issue by rodica-andronache** [Codeinwp/ti-onboarding#14]
Astra Pro's Sites Library has some nice elements we could also implement.
- the ability to add multiple different categories for each site (blog, business, free etc) and filter/search based on that too ( combined with the Page Builder categorization )
- for the cases, where no sites match the search criteria, have a message where we can ask for sites suggestions. https://prnt.sc/m4zq4g

Another idea ( not sure if possible, technical and according to GDPR ) would be to also collect the search results that don't return any result ( this way we would know what users are looking for ). @selul would that be possible?

**Comment by selul** [comment]
@rodica-andronache yes, we can do that. Is the search calling a remote api? Or it's a local search?

**Comment by rodica-andronache** [comment]
a local search
